web/src/pages/calendar/index.js

```diff
@@ -9,16 +9,18 @@ import SEO from '../../components/seo';
 const Calendar = () => {
   return (
     <Layout>
+      <main>
```

Comment:
@megansizemore would it be okay to move the `<main>` tags to the `<Layout>` file? This way, we don't have to remember to do it for each file.

Reply:
@raney24 yes, I just made those changes. thanks for catching that!